streams/src/test/java/org/apache/kafka/streams/state/internals/CompositeReadOnlyKeyValueStoreTest.java

```diff
@@ -171,6 +208,10 @@ public void shouldThrowInvalidStoreExceptionDuringRebalance() throws Exception {
         rebalancing().get("anything");
     }
 
+    @Test(expected = InvalidStateStoreException.class)
+    public void shouldThrowInvalidStoreExceptionOnApproximateNumEntriesDuringRebalance() throws Exception {
+        rebalancing().approximateNumEntries();
```

Comment:
Hmmm... Not sure. The "it might change in the future" argument is valuable. What do others think? Splitting seems to be the save option.

\cc @enothereska @bbejeck @dguy @guozhangwang